manifests/d/dbeaver/dbeaver/21.1.2/dbeaver.dbeaver.locale.en-US.yaml

```
@@ -0,0 +1,34 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: dbeaver.dbeaver
+PackageVersion: 21.1.2
+PackageLocale: en-US
+Publisher: DBeaver Corp
+Author: DBeaver Corp
+PackageName: DBeaver
+PackageUrl: https://dbeaver.io
+License: Apache License v2
+LicenseUrl: https://dbeaver.io/product/dbeaver_license.txt
+ShortDescription: dbeaver.dbeaver
```

```suggestion
ShortDescription: Free multi-platform database tool for developers, database administrators, analysts and all people who need to work with databases.
```
Issue from 0.1.0 to 1.0.0 Manifest Schema Update